setup.cfg

```diff
@@ -0,0 +1,54 @@
+[metadata]
+name = wktplot
+version = attr: wktplot.__version__
+author = Michael Simpson, Gerald Sornsen
+author_email = mikeysimpson4@gmail.com, gerald@sornsen.io
+description = Well-known-text and shapely object visualiation library
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/FuzzFoundation/WKTPlot
+platform = any
+license = MIT
+keywords = well-known-text, shapely, bokeh, geo-spatial, visualiation
+classifiers = 
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Natural Language :: English
+    Operating System :: OS Independent
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Topic :: Software Development :: Librarie
+
+[options]
+include_package_data = True
+package_dir = 
+    =src
```

Comment:
Is this extra `=` necessary?

Reply:
Yeah https://setuptools.pypa.io/en/latest/userguide/declarative_config.html?highlight=package_dir#:~:text=described%20above.%0A%0A%5Boptions%5D-,package_dir,-%3D%0A%20%20%20%20%3Dsrc%0Apackages%3Dfind